esrally/metrics.py

```diff
@@ -825,6 +828,29 @@ def _get(self, name, task, operation_type, sample_type, node_name, mapper):
         self.logger.debug("Metrics query produced [%s] results.", result["hits"]["total"])
         return [mapper(v["_source"]) for v in result["hits"]["hits"]]
 
+    def get_one(self, name, sample_type=None, node_name=None, task=None, mapper=lambda doc: doc["value"],
+                sort_key=None, sort_reverse=False):
+        order = "desc"
+        if not sort_reverse:
+            order = "asc"
+        if sort_key:
+            query = {
+                "query": self._query_by_name(name, task, None, sample_type, node_name),
+                "sort": [
+                    {sort_key: {"order": order}}
+                ],
+                "size": 1
+            }
+        else:
+            query = {
+                "query": self._query_by_name(name, task, None, sample_type, node_name),
+                "size": 1
+            }
+        self.logger.debug("Issuing get against index=[%s], query=[%s].", self._index, query)
+        result = self._client.search(index=self._index, body=query)
+        self.logger.debug("Metrics query produced [%s] results.", result["hits"]["total"])
+        return mapper(result["hits"]["hits"][0]["_source"])
```

Comment:
This would fail with a key error if there are no hits. Can we check this and return `None` if there are no hits?